plugin-interface/src/main/scala/mesosphere/marathon/plugin/task/AppTaskProcessor.scala

```diff
@@ -1,13 +1,30 @@
 package mesosphere.marathon.plugin.task
 
 import mesosphere.marathon.plugin.AppDefinition
-import mesosphere.marathon.plugin.plugin
-import org.apache.mesos.Protos
+import mesosphere.marathon.plugin.plugin.Plugin
+import org.apache.mesos.Protos.TaskInfo
 
 /**
   * AppTaskProcessor is a factory func that generates functional options that mutate Mesos
```

Comment:
doh!

On Thu, May 12, 2016 at 9:53 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In
> plugin-interface/src/main/scala/mesosphere/marathon/plugin/task/AppTaskProcessor.scala
> <https://github.com/mesosphere/marathon/pull/3856#discussion_r63023600>:
>
> > @@ -1,13 +1,30 @@
> >  package mesosphere.marathon.plugin.task
> >
> >  import mesosphere.marathon.plugin.AppDefinition
> > -import mesosphere.marathon.plugin.plugin
> > -import org.apache.mesos.Protos
> > +import mesosphere.marathon.plugin.plugin.Plugin
> > +import org.apache.mesos.Protos.TaskInfo
> >
> >  /**
> >    * AppTaskProcessor is a factory func that generates functional options that mutate Mesos
>
> Why "options"? Is this up-to-date?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/marathon/pull/3856/files/c456c164c2423a0824167bdb773def6844b7c066..c042ec922e21733288f8193e72cf4027bb5f18bd#r63023600>
>